assets/css/pets-love.css

```diff
@@ -0,0 +1,48 @@
+.pets-love-wrapper{
+    height:70vh;
+    width:100%;
+    display:flex;
+}
+.pets-love-heading{
+    width:33.33%;
+    height:80%;
```

Comment:
Coś mi tutaj nie pasuje. Nie ma sensu ustawiania tego height. 
Nie jestem pewien czy dobrze myślę, bo nie sprawdzam jak to wygląda - ale usuń height i daj na pets-love-wrapper 

"align-items: center;"